vrk-kpa/suomidigi
# SUOPA-428 Event content type

To test: 
- `make fresh`
- Log in and see if you can add events from https://suomidigi.docker.sh/node/add/event
    - Note. Doesn't include theming as it will change in near future..
